src/core/p5.Renderer2D.js

```diff
@@ -354,54 +354,92 @@ define(function(require) {
   // SHAPE | 2D Primitives
   //////////////////////////////////////////////
 
+  /**
+   * Generate a cubic Bezier representing an arc on the unit circle of total
+   * angle `size` radians, beginning `start` radians above the x-axis. Up to
+   * four of these curves are combined to make a full arc.
+   *
+   * See www.joecridge.me/bezier.pdf for an explanation of the method.
```

Comment:
Wait, did you find this bug because you'd written a paper about this, or did you write a paper as part of fixing this bug?